Doc/library/dataclasses.rst

```diff
@@ -491,6 +491,22 @@ depend on one or more other fields.  For example::
         def __post_init__(self):
             self.c = self.a + self.b
 
+If the dataclass inherits from another class, which defines an :meth:`__init__` 
```

Comment:
I'd like to start with something like: "The `__init__` function generated by `@dataclass` does not call base class `__init__` methods. If base class `__init__` methods need to be called, it's common to do so in a `__post_init__` method."

Reply:
Thanks for the feedback! I have adapted this text slightly in my last commit to reflect that the `__init__` ought to be called if the base class has a non-default `__init__` method, i.e. the one defined on `object` (this is also mentioned [here in the docs](https://docs.python.org/3/reference/datamodel.html?highlight=__init__#object.__init__)). I was wondering if it should be mentioned that, when dealing with `dataclasses`, this is likely the case (I think that the generation of `__init__` by `@dataclass` might obscure the presence of a non-default `__init__`).